docs/docs/tutorials/cost-estimate-report.md

```diff
@@ -0,0 +1,116 @@
+---
+id: cost-estimate-report
+title: Create a cost estimate report
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Infracost?
+
+[Infracost](https://www.infracost.io/) is a tool that enables developers to analyze their cloud cost estimates using their Terraform templates. Nebari wraps the `infracost` command-line interface, which is free and open-source, to provide cost estimates for your Nebari deployments across major cloud providers from your terminal.
+
+This tutorial will generate a cost report for our Nebari deployment, which you can share with other users and groups via a simple Infracost link. We will also be looking into how Infracost can be configured on your local machine. 
+
+Infracost natively supports Amazon Web Services, Azure, and Google Cloud Platform. For this tutorial, we will be using the Google Cloud Platform (GCP) deployment to generate the cost report.
+
+## Installing Infracost
+
+
+To install the `infracost` command-line interface, run the following command on your terminal:
+
+```sh
+curl -fsSL https://raw.githubusercontent.com/infracost/infracost/master/scripts/install.sh | sh
+```
+
+The script downloads the the CLI based on your operating-system/architecture and puts it in `/usr/local/bin`.
+
+After installation, you can run `infracost` locally via the following command:
+
+```sh
+infracost --version
+```
+
+Next, you'll need to configure the API key for Infracost to enable generating cost reports and dashboards. Nebari's cost estimate feature requires this API key. To generate the key, run the following command on your terminal:
+
+```sh
+infracost register
+```
+
+Now you can run the following command to validate your API key:
+
+```sh
+infracost configure get api_key
+```
+
+We are now ready to run the cost estimate tool.
+
+## Generating the cost estimates
+
+To generate the cost estimate, run the following command:
+
+```sh
+nebari cost-estimate
+```
+
+The `nebari cost-estimate` will look for the `stages` directory and analyze the Terraform templates to generate a cost report. It will consist of two tables describing the cost and resources breakdown, a publicly-accessible Infracost dashboard URL and additional documentation on the edge cases that the cost estimate tool currently misses.
```

Comment:
```suggestion
The `Nebari cost estimate` tool will look for the `stages` directory within your deployment file structure and analyze the Terraform templates to generate a cost report. It will consist of two tables describing the cost and resources breakdown, a publicly-accessible Infracost dashboard URL and additional documentation on the edge cases that the cost estimate tool currently misses.
```

Reply:
is this accurate?